### PR TITLE
fix(treesitter): remove virtual text from playground

### PR DESCRIPTION
**Pull request by lewis6991**
Implement the range and lang annotations as comments instead

### CODE REVIEW

**Comment by lewis6991** [comment]
@clason 

image

**Comment by justinmk** [comment]
What problem is this solving?

**Comment by lewis6991** [comment]
https://github.com/neovim/neovim/pull/22191

@clason wants to be able to yank text in the tree with the ranges and language.

File: runtime/lua/vim/treesitter/playground.lua [hunk 1]
```diff
@@ -138,33 +138,50 @@ end
 
 local decor_ns = api.nvim_create_namespace('ts.playground')
 
+---@param lnum integer
+---@param col integer
+---@param end_lnum integer
+---@param end_col integer
+---@return string
+local function get_range_str(lnum, col, end_col, end_lnum)
```
Comment (clason) [inline runtime/lua/vim/treesitter/playground.lua:146]: needs to be marked `---@private`?

**Review by clason — APPROVED** [review]
Thanks!

**Pull request merged**